Jenkinsfile

```diff
@@ -0,0 +1,125 @@
+@Library('github.com/connexta/cx-pipeline-library@master') _
+@Library('github.com/connexta/github-utils-shared-library@master') __
+
+pipeline {
+    agent {
+        node {
+            label 'linux-small'
+            customWorkspace "/jenkins/workspace/${JOB_NAME}/${BUILD_NUMBER}"
+        }
+    }
+    options {
+        buildDiscarder(logRotator(numToKeepStr:'25'))
+        disableConcurrentBuilds()
+        timestamps()
+        skipDefaultCheckout()
+    }
+    triggers {
+        /*
+          Restrict nightly builds to master branch, all others will be built on change only.

```

Comment:
❓ Is this comment correct? Looks like this runs weekly if it is master versus nightly.

Reply:
i can ask @dcruver about frequency